<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubmoduleMainWindow</class>
 <widget class="QMainWindow" name="SubmoduleMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guitar</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="RepositoryWrapperFrame" name="frame_repository_wrapper2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter_h">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="LogTableWidget" name="tableWidget_log2">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="tabKeyNavigation">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
         <widget class="QStackedWidget" name="stackedWidget_filelist">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_uncommited">
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,1">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="FilesListWidget" name="listWidget_unstaged">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_2">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>3</number>
                 </property>
                 <item row="2" column="0" colspan="2">
                  <widget class="Line" name="line">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QToolButton" name="toolButton_unstage">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>64</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                   </property>
                   <property name="text">
                    <string>Unstage</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/resources.qrc">
                     <normaloff>:/image/unstage.svg</normaloff>:/image/unstage.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextUnderIcon</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0" colspan="2">
                  <widget class="QToolButton" name="toolButton_select_all">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                   </property>
                   <property name="text">
                    <string>Select all</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/resources.qrc">
                     <normaloff>:/image/selall.svg</normaloff>:/image/selall.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>24</width>
                     <height>18</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextUnderIcon</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QToolButton" name="toolButton_stage">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>64</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                   </property>
                   <property name="text">
                    <string>Stage</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/resources.qrc">
                     <normaloff>:/image/stage.svg</normaloff>:/image/stage.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextUnderIcon</enum>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0" colspan="2">
                  <widget class="QToolButton" name="toolButton_commit">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="focusPolicy">
                    <enum>Qt::StrongFocus</enum>
                   </property>
                   <property name="text">
                    <string>Commit</string>
                   </property>
                   <property name="icon">
                    <iconset resource="resources/resources.qrc">
                     <normaloff>:/image/commit.svg</normaloff>:/image/commit.svg</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>24</width>
                     <height>24</height>
                    </size>
                   </property>
                   <property name="toolButtonStyle">
                    <enum>Qt::ToolButtonTextUnderIcon</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="FilesListWidget" name="listWidget_staged">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_files">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="FilesListWidget" name="listWidget_files">
              <property name="contextMenuPolicy">
               <enum>Qt::CustomContextMenu</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QFrame" name="frame_diff_view">
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="FileDiffWidget" name="widget_diff_view" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>816</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FileDiffWidget</class>
   <extends>QWidget</extends>
   <header>FileDiffWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogTableWidget</class>
   <extends>QTableWidget</extends>
   <header>LogTableWidget.h</header>
  </customwidget>
  <customwidget>
   <class>FilesListWidget</class>
   <extends>QListWidget</extends>
   <header>FilesListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>RepositoryWrapperFrame</class>
   <extends>QFrame</extends>
   <header>RepositoryWrapperFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableWidget_log2</tabstop>
  <tabstop>listWidget_unstaged</tabstop>
  <tabstop>toolButton_select_all</tabstop>
  <tabstop>toolButton_stage</tabstop>
  <tabstop>toolButton_unstage</tabstop>
  <tabstop>toolButton_commit</tabstop>
  <tabstop>listWidget_staged</tabstop>
  <tabstop>listWidget_files</tabstop>
  <tabstop>widget_diff_view</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
